src/models/vamp.jl

```diff
@@ -0,0 +1,96 @@
+export VAMP, mmd_mean_vamp, init_vamp_mean, init_vamp_sample
+import ConditionalDists: rand # so that there is no conflict
+"""
+	VAMP{K<:Int,P<:AbstractArray}(K::Int, xdim::Union{Tuple, Int})
+	VAMP{K<:Int,P<:AbstractArray}(X::P)
+
+Vamp prior with `K` pseudoinputs `P`. If only the pseudoinput `X` is given, `K` is the size of 
+the last dimension. 
+
+# Arguments
+* `K`: Number of pseudoinputs
+* `xdim`: Size of an individual pseudoinput
```

Comment:
is xdim used somehwere?

Reply:
Yes, in the constructor where you only supply the size of a single pseudoinput `xdim` and the number of them that you want to use `K`.

Reply:
ahh, yes I see it now :)